src/main/scala/firrtl/DependencyAPIMigration.scala

```diff
@@ -15,6 +15,10 @@ import firrtl.stage.TransformManager.TransformDependency
   *
   * For more information, see: https://bit.ly/2Voppre
   */
+@deprecated(
+  "DependencyAPIMigration will finish its job in FIRRTL 1.5, and it will be removed in FIRRTL 1.6",
+  "FIRRTL 1.5"
+)
```

Comment:
I'd add a "Please remove DependencyAPIMigration from your Transform to be compatible with FIRRTL 1.6." (Or something along those lines.)